pcdsdevices/gasdet.py

```diff
@@ -0,0 +1,345 @@
+        The default '202' applies to the HXR gas detector system.
+    module : ``str``
+        3-digit number representing the digitizer board,
+        e.g. '240', '360'.
+
+    """
+    ch1 = FCpt(
+        acqiris_channel,
+        '{self.prefix}:{self._mod_pref}1',
+        kind='hinted'
+    )
+    ch2 = FCpt(
+        acqiris_channel,
+        '{self.prefix}:{self._mod_pref}2',
+        kind='hinted')
+    ch3 = FCpt(
+        acqiris_channel,
+        '{self.prefix}:{self._mod_pref}3',
+        kind='hinted')
+    ch4 = FCpt(
+        acqiris_channel,
+        '{self.prefix}:{self._mod_pref}4',
+        kind='hinted')
+
+    tab_component_names = True
+    tab_whitelist = ["ch1", "ch2", "ch3", "ch4"]
+
+    run_state = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MDAQStatus',
+        kind='config',
+        string=True
+    )
+    sample_interval = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MSampInterval',
+        kind='config'
+    )
+    delay_time = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MDelayTime',
+        kind='config'
+    )
+    num_samples = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MNbrSamples',
+        kind='config'
+    )
+    trig_source = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MTriggerSource',
+        kind='config', string=True)
+    trig_class = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MTrigClass',
+        kind='config',
+        string=True
+    )
+    samples_freq = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MSampFrequency',
+        kind='config'
+    )
+    rate = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MEventRate',
+        kind='config'
+    )
+    num_events = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MEventCount',
+        kind='config'
+    )
+    num_trig_timeouts = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MTriggerTimeouts',
+        kind='config'
+    )
+    num_truncated = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MTruncated',
+        kind='config'
+    )
+    acq_type = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MType',
+        kind='config',
+        string=True
+    )
+    acq_mode = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MMode',
+        kind='config',
+        string=True
+    )
+    acq_mode_flags = FCpt(
+        EpicsSignal,
+        '{self.prefix}:{self.module}:MModeFlags',
+        kind='config',
+        string=True
+    )
+    clock_type = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MClockType',
+        kind='config',
+        string=True
+    )
+    crate_number = FCpt(
+        EpicsSignalRO,
+        '{self.prefix}:{self.module}:MCrateNb',
+        kind='config'
+    )
+    nADC_bits = FCpt(
```

Comment:
Should go for naming consistency on components:
```suggestion
    n_adc_bits = FCpt(
```